### PR TITLE
Feature/add popup if no images

### PR DESCRIPTION
**Pull request by akshuga**
## Proposed Changes

  - show QMessageBox if there is no images in dataset dirs when starting training.

### CODE REVIEW

File: src/main/python/view/dataset.py [hunk 1]
```diff
@@ -168,6 +168,26 @@ def on_clicked_delete_images_button(self):
             self._reload_images(self.__selected_dataset_category())
 
     def on_clicked_train_button(self):
+
+        if not [img for img in os.listdir(Dataset.images_path(Dataset.Category.TEST_NG)) if
+                     Path(img).suffix in ['.jpg', '.jpeg', '.png', '.gif', '.bmp']]:
```
Comment (jrsaruo) [inline src/main/python/view/dataset.py]: @akshuga 
`['.jpg', '.jpeg', '.png', '.gif', '.bmp']` should be reused as a local variable in lines 173, 179 and 185.

File: src/main/python/view/dataset.py [hunk 2]
```diff
@@ -168,6 +168,26 @@ def on_clicked_delete_images_button(self):
...
+            self.msgBox = QMessageBox()
+            self.msgBox.setText('性能評価用の不良品画像フォルダが空です.\nトレーニングを開始するには不良品画像を1枚以上を追加してください.')
```
Comment (jrsaruo) [inline src/main/python/view/dataset.py]: @akshuga 
「画像を1枚以上 **を** 追加」 => 「画像を1枚以上追加」

File: src/main/python/view/dataset.py (hunk 1 above)
Comment (akshuga) [reply src/main/python/view/dataset.py]: add local variable "img_suffix_list" in on_clicked_train_button

File: src/main/python/view/dataset.py (hunk 2 above)
Comment (akshuga) [reply src/main/python/view/dataset.py]: FIxed.

**Review by jrsaruo — APPROVED** [review]

**Pull request merged**